update-tests/testall.sh

```diff
@@ -2,7 +2,8 @@
 export LC_ALL=C
 
 proj=$1
-(cd ../osc/openSUSE\:$proj/_product/ && osc up )
+product=$2
+(cd ../osc/openSUSE\:$proj/$product/ && osc up )
```

Comment:
nit: might as well fix the unbalanced space as the new one introduced lacks it as well.